src/style/custom-properties/colors.css

```diff
@@ -32,7 +32,7 @@
   --uui-color-space-cadet: $space-cadet;
   --uui-color-space-cadet-light: color($space-cadet lightness(+ 8%));
   --uui-color-space-cadet-dark: color($space-cadet blackness(+ 8%));
-  --uui-color-space-cadet-dimmed: color($space-cadet saturation(- 30%));
+  --uui-color-space-cadet-dimmed: color($space-cadet saturation(- 30%) lightness(+ 30%));
```

Comment:
And a fixed color for the disabled state, which is still used for first, previous, next, last buttons.